package-lock.json

```diff
@@ -21365,6 +21320,12 @@
             "glob": "^7.1.3"
           }
         },
+        "semver": {
```

Comment:
`semver` is still here, they are persistent 😄

Reply:
Yes, it should be there. With out recent addition of `commitlint`, the pakcage-lock is wrong.